ui_testing/pages/order_page.py

```diff
@@ -332,21 +332,23 @@ def get_suborder_data(self):
             "suborders": []
```

Comment:
Why you update this function, you didn't use it in the test case

Reply:
I made this branch as branch of update material_type test case as I needed functions in it, So any changes in  update material_type branch will be added here.